src/backend/gog/games.ts

```diff
@@ -672,42 +674,58 @@ class GOGGame extends Game {
 
     const [object] = array.splice(index, 1)
     logInfo(['Removing', object.install_path], LogPrefix.Gog)
-    // TODO: Run unins000.exe /verysilent /dir=Z:/path/to/game
+    // Run unins000.exe /verysilent /dir=Z:/path/to/game
     const uninstallerPath = join(object.install_path, 'unins000.exe')
 
     const res: ExecResult = { stdout: '', stderr: '' }
     if (existsSync(uninstallerPath)) {
-      const {
-        winePrefix,
-        wineVersion: { bin, name },
-        wineCrossoverBottle
-      } = GameConfig.get(this.appName).config
-      let commandPrefix = `WINEPREFIX="${winePrefix}" ${bin}`
-      if (name.includes('CrossOver')) {
-        commandPrefix = `CX_BOTTLE=${wineCrossoverBottle} ${bin}`
-      }
-      const command = `${
-        isWindows ? '' : commandPrefix
-      } "${uninstallerPath}" /verysilent /dir="${isWindows ? '' : 'Z:'}${
-        object.install_path
-      }"`
-      logInfo(['Executing uninstall command', command], LogPrefix.Gog)
-      execAsync(command)
-        .then(({ stdout, stderr }) => {
-          res.stdout = stdout
-          res.stderr = stderr
-        })
-        .catch((error) => {
-          res.error = `${error}`
+      const gameSettings = GameConfig.get(this.appName).config
+
+      const installDirectory = isWindows
+        ? object.install_path
+        : await getWinePath({
+            path: object.install_path,
+            gameSettings
+          })
+
+      const command = [
+        uninstallerPath,
+        '/verysilent',
+        `/dir=${shlex.quote(installDirectory)}`
+      ]
+
+      logInfo(['Executing uninstall command', command.join(' ')], LogPrefix.Gog)
+
+      if (!isWindows) {
+        runWineCommand({
+          gameSettings,
+          commandParts: command,
+          wait: true,
+          protonVerb: 'waitforexitandrun'
         })
+      } else {
+        const adminCommand = [
+          'Start-Process',
+          '-FilePath',
+          uninstallerPath,
+          '-Verb',
+          'RunAs',
+          '-ArgumentList'
+        ]
+
+        await spawnAsync('powershell', [
```

Comment:
maybe we can use `spawnSync` instead of `await spawnAsync`

Reply:
I disagree with that. I added this method for us to have a way of having the same behavior of `execAsync,  by using await at least we know that nothing else will be executed before it finishes. I think that's the intention here.